integration/resource_lacework_integration_service_now_test.go

```diff
@@ -0,0 +1,50 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestServiceNowRestAlertChannelCreate applies integration terraform '../examples/resource_lacework_alert_channel_service_now'
+// Uses the go-sdk to verify the created integration
+// Applies an update with new channel name and Terraform destroy
+func TestServiceNowRestAlertChannelCreate(t *testing.T) {
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/resource_lacework_alert_channel_service_now",
+		Vars: map[string]interface{}{
+			"channel_name":         "Service Now Alert Channel Example",
+			"instance_url":         "https://dev123.service-now.com",
+			"username":             "snow-user",
+			"password":             "snow-pass",
+			"custom_template_file": customTemplate,
+		},
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Create new ServiceNowRest Alert Channel
+	create := terraform.InitAndApplyAndIdempotent(t, terraformOptions)
+	created := GetAlertChannelProps(create)
+
+	assert.Equal(t, "Service Now Alert Channel Example", created.Data.Name)
```

Comment:
can we assert all the above values before the next apply? Something like we do below but here: